### DISCUSSION TITLE
Do not return JS scripts as log output

### DISCUSSION THREAD

**Discussion by PascalTurbo** [aquasecurity/trivy#5117] [Bugs]
### Description

When trivy detects an issue in a js file ("LOW: LinkedIn (linkedin-client-id)" f.e.), it returns the line where the issue occurs. While this might be a good behavior for normal JS files, it's a problem with minimized JS files as trivy will log the whole file.
This leads to large log files (sometimes > 50MB) that are hard to parse.

### Desired Behavior

Disable JS output via cli param or trim the output of source files after x characters.

### Actual Behavior

Large log files

### Reproduction Steps

```bash
I can not share our source code. But if you have test cases with JS just add huge minified JS code.
```

### Target

Container Image

### Scanner

None

### Output Format

None

### Mode

Standalone

### Debug Output

```bash
Not necessary
```

### Operating System

Linux / Docker

### Version

```bash
0.44.1
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by itaysk** [comment]
trimming sounds like a good idea 👍
